src/pgrn-compatible.h

```diff
@@ -73,3 +75,21 @@ typedef Oid PGrnRelFileNumber;
 		((relkind) == RELKIND_RELATION || (relkind) == RELKIND_TOASTVALUE ||   \
 		 (relkind) == RELKIND_MATVIEW)
 #endif
+
+static inline IndexScanDesc
+pgrn_index_beginscan(Relation heapRelation,
+					 Relation indexRelation,
+					 Snapshot snapshot,
+					 int nKeys,
+					 int nOrderBys)
+{
+
+	return index_beginscan(heapRelation,
+						   indexRelation,
+						   snapshot,
+#if PG_VERSION_NUM >= 180000
+						   NULL,
```

Comment:
Do you think we should always set NULL?
Do we need to support show index search count?

Reply:
This information is used to display when EXPLAIN ANALYZE.
If we wanted to display additional information there, we could implement it.
First of all, I think the same behavior as before is ok, so I have set NULL.

Reply:
I see.
How about the above comment is written to commit message also.